### PR TITLE
need advice

### PR DESCRIPTION
**Pull request by josselinchevalay**
hello my propal is create a gradle plugin to licenseFinder

### CODE REVIEW

File: src/test/groovy/com/ullink/gradle/entity/DependenciesDecisionEntitySpec.groovy [hunk 1]
```diff
@@ -4,6 +4,8 @@ import spock.lang.Specification
 
 class DependenciesDecisionEntitySpec extends Specification {
     def 'create Dependencies Decision entity'() {
+        given:
```
Comment (vincent-vandemeulebrouck-ullink) [inline src/test/groovy/com/ullink/gradle/entity/DependenciesDecisionEntitySpec.groovy]: minor: should not be needed

File: src/main/groovy/com/ullink/gradle/entity/DependenciesDecisionEntity.groovy [hunk 2]
```diff
@@ -5,14 +5,14 @@ class DependenciesDecisionEntity {
     String version
     String shortLicenseName
     String homepage
-    boolean approove
+    boolean approve
```
Comment (vincent-vandemeulebrouck-ullink) [inline src/main/groovy/com/ullink/gradle/entity/DependenciesDecisionEntity.groovy]: used in MakeDecisionTask.groovy
+ descision to rename to decision ;-)

Comment (josselinchevalay) [reply src/main/groovy/com/ullink/gradle/entity/DependenciesDecisionEntity.groovy]: done

File: src/test/groovy/com/ullink/gradle/entity/DependenciesDecisionEntitySpec.groovy (hunk 1 above)
Comment (josselinchevalay) [reply src/test/groovy/com/ullink/gradle/entity/DependenciesDecisionEntitySpec.groovy]: done

**Pull request merged**